### DISCUSSION TITLE
Export / Import labels.txt on Windows standalone version

### DISCUSSION THREAD

**Discussion by abundis-rmn2** [wkentaro/labelme#1048] [Q&A / Help] [answered]
Hi, am running the standalone version of labelme, until now is working like a charm. The thing is i cant run it from cmd to specify the label.txt file.

So any time i open a folder already in tagging process, the labels list is empty and i have to go trough some already tagged images and the list will populate with labels found.

The thing here is, is there anyway to export this labels.txt file? 
An then, import it in windows standalone version?

![image](https://user-images.githubusercontent.com/67477259/175719332-f7f33e3a-3170-4a14-8fa8-5bd972ff1c67.png)

If this is not possible, how should i make this file, for training just:
Caracter
Tag
Bomba
Roller
Wildsytle
3D
Moniker
S_Tren

Thanks for the help.

**Comment by abundis-rmn2** [comment]
What about:
__ignore__
_background_

Those should be include?

**Comment by wkentaro** [comment, marked as answer]
You need to pass the labels by `labelme --labels Caracter,Tag,Bomba,Roller,...` or `labelme --labels labels.txt`.

**Reply by abundis-rmn2** [reply]
Thanks a lot, is a standalone version, i handmade the labels.txt file and work like a charm.

**Reply by uasdev** [reply]
Hi @wkentaro @abundis-rmn2 ,
I am trying to run `labelme2voc.py` (for the object detection example), but I have no idea how to generate a labels.txt file for it.
I am trying to export the labels.txt directly from labelme GUI.
I went through all my images and labeled them with different classes. How do  export the labels as label.txt? If not, could you provide an example of what the labels.txt file should look like?
